### PR TITLE
Feat/lib 388 Tab

### PR DESCRIPTION
**Pull request by Crish15**
Jira: [LIB-388](https://fiscozen.atlassian.net/browse/LIB-388)

### CODE REVIEW

File: apps/storybook/tailwind.config.js [hunk 1]
```diff
@@ -3,5 +3,9 @@ import tailwindConfig from '@fiscozen/style/tailwind.config';
 /** @type {import('tailwindcss').Config} */
 export default {
     ...tailwindConfig,
-    content: ["./src/**/*.{html,js,ts,vue}", "./node_modules/@fiscozen/**/src/**/*.{html,js,ts,vue}"],
+    content: [
+        "./src/**/*.{html,js,ts,vue}", 
+        "./node_modules/@fiscozen/**/src/**/*.{html,js,ts,vue}",
+        "./node_modules/@fiscozen/tab/src/**/*.{html,js,ts,vue}"
```
Comment (Crish15) [inline apps/storybook/tailwind.config.js:9]: rimuovere

File: packages/tab/src/FzTabs.vue [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+provide("selectedTab", selectedTab);
+
+const tabs = computed(() => {
+  if (!slots.default) return [];
+
+  return slots
+    .default()
+    .filter((tab) => {
+      // @ts-ignore vue3 types are not updated
+      return tab.type && tab.type.__name === "FzTab";
```
Comment (Crish15) [inline packages/tab/src/FzTabs.vue:35]: remove ts-ignore

File: packages/tab/src/__test__/FzTabs.test.ts
Comment (Crish15) [inline packages/tab/src/__test__/FzTabs.test.ts:1]: add test on click

File: packages/tab/src/FzTabs.vue [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+function onWheel(e: WheelEvent) {
+  e.preventDefault();
+  e.stopPropagation();
+  if (e.deltaY > 0) tabContainer.value!.scrollLeft += 100;
+  else tabContainer.value!.scrollLeft -= 100;
+}
+
+onMounted(() => {
+  console.log("entro")
```
Comment (oxcened) [inline packages/tab/src/FzTabs.vue:55]: log

File: packages/tab/src/FzTabs.vue [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    selectedTab.value = findSelected.title;
+  } else {
+    selectedTab.value = tabs.value[0].title;
+  }
+
+  const duplicateTitles = tabs.value
+    .map((tab) => tab.title)
+    .filter((title, index, self) => self.indexOf(title) !== index);
+  
+  if(duplicateTitles.length > 0) {
```
Comment (oxcened) [inline packages/tab/src/FzTabs.vue:72]:
```suggestion
  if (duplicateTitles.length) {
```

File: packages/tab/src/FzTabPicker.vue [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        <button 
+          v-for="tab in tabs" 
+          :key="tab.title" 
+          :size="size"
+          @click="() => {isOpen = false; selectedTab = tab.title}" 
+          :class="[
+            computedClasses,
+            selectedTab === tab.title ? '!bg-background-alice-blue' : 'hover:!bg-background-alice-blue !text-black hover:!text-blue-500',
+            tab.disabled ? 'cursor-not-allowed' : 'cursor-pointer'
+          ]"
```
Comment (oxcened) [inline packages/tab/src/FzTabPicker.vue:24]: I would keep everything in `computedClasses`

File: packages/tab/src/FzTabPicker.vue [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+            <span class="overflow-hidden text-ellipsis whitespace-nowrap"> {{ selectedTab }}  </span>
+            <FzIcon :name="isOpen ? 'chevron-up':'chevron-down'" :size="size"/>
+          </button>
+      </template> 
+      <div class="flex flex-col p-4 rounded-[4px] shadow overflow-hidden" :style="{ width: containerWidth }">
+        <button 
+          v-for="tab in tabs" 
+          :key="tab.title" 
+          :size="size"
+          @click="() => {isOpen = false; selectedTab = tab.title}"
```
Comment (oxcened) [inline packages/tab/src/FzTabPicker.vue:19]: I would move this to a function

**Review by Trebh — APPROVED** [review]

**Review by oxcened — APPROVED** [review]

**Pull request merged**